index.html

```diff
@@ -1,13 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This a portfolio of a full stack web developer">
+    <meta name="keywords" content="porfotlio, software developer, web developer">
+    <meta name="author" content="Julio Quezada">
     <link rel="stylesheet" href="./assets/css/style.css">
     <title>Document</title>
 </head>
+
 <body>
-    <h1 class="makeHeaderRed">My portfolio</h1>
+    <header class="header-container">
+        <div class="logo">
+            <p class="main-logo">John Doe</p>
+        </div>
+        <nav>
+            <div class="hamburgerMenu">
+                <ul class="nav-items">
+                    <li class="list-item"><a href="#">Porfolio</a></li>
+                    <li class="list-item"><a href="#">About</a></li>
+                    <li class="list-item"><a href="#">Contact</a></li>
+                    <li class="list-item"><a href="#"><img src="./assets/images/mail.png" alt="mail logo"></a></li>
+                </ul>
+            </div>
+            <button type="button" class="hamburger-icon">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </button>
+        </nav>
+    </header>
+
+    <main class="main-area">
+        <section class="main-section">
+            <div class="salute">
+                <h1 class="main-header"> Hey There.<span class="subtitle">I'm Alex</span></h1>
```

Comment:
[OPTIONAL] 👍🏽 
- In my opinion, I think it will be better if you used your name instead, because the portfolio is about YOUR works and skills.  Maybe just your first name will be fine. What do you think? 🤔